src/Administration/Resources/app/administration/src/module/sw-order/view/sw-order-create-initial/sw-order-create-initial.spec.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @package checkout
+ */
+
+import { mount } from '@vue/test-utils';
+import orderStore from 'src/module/sw-order/state/order.store';
+
+const responses = global.repositoryFactoryMock.responses;
+
+responses.addResponse({
+    method: 'Post',
+    url: '/search/customer',
+    status: 200,
+    response: {
+        data: [
+            {
+                id: '1234',
+                attributes: {
+                    id: '1234',
+                },
+                relationships: [],
+            },
+        ],
+    },
+});
+
+async function createWrapper(customerId = null) {
+    return mount(await wrapTestComponent('sw-order-create-initial', { sync: true }), {
+        global: {
+            mocks: {
+                $route: {
+                    query: {
+                        customerId: customerId,
+                    },
+                },
+            },
+        },
+    });
+}
+
+describe('src/module/sw-order/view/sw-order-create-initial', () => {
+    let wrapper;
+
+    beforeEach(async () => {
+        wrapper = await createWrapper();
+
+        if (Shopware.State.get('swOrder')) {
+            Shopware.State.unregisterModule('swOrder');
+        }
+
+        Shopware.State.registerModule('swOrder', orderStore);
+
+        await flushPromises();
+    });
+
+    afterEach(async () => {
+        if (wrapper) {
+            await wrapper.unmount();
+        }
+
+        await flushPromises();
+    });
+
+    it('should be a Vue.js component', async () => {
+        expect(wrapper.vm).toBeTruthy();
+    });
+
+    it('should not load a customer if no customerId query parameter has been passed', async () => {
+        wrapper = await createWrapper();
+        wrapper.vm.customerRepository.get = jest.fn();
+        await flushPromises();
+
+        expect(wrapper.vm.customerRepository.get).not.toHaveBeenCalled();
```

Comment:
As the `jest.fn()` is added after the wrapper has been created, the `created` method has already run, so you need to call `wrapper.vm.created()` manually to avoid a false positive:
```suggestion
        wrapper = await createWrapper();
        await flushPromises();

        wrapper.vm.customerRepository.get = jest.fn();
        wrapper.vm.created();
        await flushPromises();

        expect(wrapper.vm.customerRepository.get).not.toHaveBeenCalled();
```

Reply:
Ok thanks! I thought the “created” function is only triggered by the “flushPromises”. 

I have adjusted it. However, I got the error that “wrapper.vm.created()” is not a function. That's why I've now used “createdComponent”.